index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Microverse Porfolio Setup</title>
+    <link rel="stylesheet" href="main.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+  </head>
+  <body>
+   
```

Comment:
- [ ] [REQUIRED] Please, use the `Poppins` font for the heading `<h1>` to match the design as shown in the picture below ⬇️.
- [ ] [REQUIRED] Also, you can adjust the size of the social icons ♻️ to match their counter parts in the original design 🖼️ .

Your website | The figma design
-----|-----
![Web capture_21-4-2022_327_rawcdn githack com](https://user-images.githubusercontent.com/60979926/164351144-56790aba-c63f-4981-8db0-ca8d6c4dcd9d.jpeg) | ![image](https://user-images.githubusercontent.com/60979926/164351706-a670e055-90ce-49d2-b2a8-96c15d2ad130.png)

Reply:
Poppin applied to the h1